llvm/lib/SandboxIR/SandboxIR.cpp

```diff
@@ -575,6 +575,33 @@ void Instruction::dumpOS(raw_ostream &OS) const {
 }
 #endif // NDEBUG
 
+FenceInst *FenceInst::create(AtomicOrdering Ordering, BBIterator WhereIt,
+                             BasicBlock *WhereBB, Context &Ctx,
```

Comment:
Why not use llvm::InsertPosition to define where to insert?

Reply:
We don't have an InsertPosition class yet. We will refactor all create() functions at some point to use that once it's available.